GoogleForCreators/web-stories-wp
# Page templates: allow removing saved templates

## Summary

- Adds the option to delete saved templates.


## Relevant Technical Choices



## To-do



## User-facing changes
N/A


## Testing Instructions
- Enable the custom page templates experiment.
- Switch to saved templates.
- Add an element to the page and Save it as template.
- Hover on the Saved template, verify the Trash icon appears as in Figma
- Verify that clicking on the Trash icon asks for confirmation and then deletes the Saved template.


### QA
See testing instructions above.





This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7062
